lib/zone-spec/fake-async-test.ts

```diff
@@ -35,7 +36,7 @@
 
     scheduleFunction(
         cb: Function, delay: number, args: any[] = [], isPeriodic: boolean = false,
-        id: number = -1): number {
+        isRequestAnimationFrame: boolean = false, id: number = -1): number {
```

Comment:
@vikerman , should we also add an option flag in `FakeAsyncTestSpec` to control consider `RAF` as timer or not?

```javascript
constructor(namePrefix: string, isRequestAnimationFrame: boolean = false) {
      this.name = 'fakeAsyncTestZone for ' + namePrefix;
      this.isRequestAnimationFrame = isRequestAnimationFrame;
    }
```